### ISSUE TITLE
Add inter-rater agreement to statistics and raw output

### ISSUE THREAD

**Issue by LizMcQuillan** [doccano/doccano#1846]
Feature description
---------
Add inter-rater agreement metrics both at the label level and in aggregate.

Use Case
----------
For every project where I have more than 1 annotator, I compute inter-rater agreement at multiple levels. Having this in Doccano would save a bit of repetitive hassle. How I envision this working is that at the label level, we can see whether two or more annotators agree on a given label both for a single datapoint and a percentage aggregation of agreement between all annotators.

Currently, I need to export data from Doccano and reformat all of the data to even try to compute this. For example, with sequence labeling, Doccano records annotators in the order an annotator clicks the sequence on the screen and so in situations with multiple labels they are out of order making it so that I can't merely look for a match or no match in the list of labels. I need to parse every label and compute the agreement from there.

Even a basic percentage match (e.g. annotators agree for 64% of labels) would be a big improvement from nothing.

`Env: MacOS`

**Comment by Ryan-LLM** [comment]
Agree, the data quality is essential which can be controlled during annotation and approval. @Hironsan Do you have any updates on this feature request?
